pylhc_submitter/constants/autosix.py

```diff
@@ -8,66 +8,76 @@
 :author: jdilly
 
 """
+from dataclasses import dataclass, fields, MISSING
 from pathlib import Path
+from typing import Union
 
-from pylhc_submitter.constants.external_paths import SIXDESK_UTILS, MADX_BIN
-from enum import IntEnum
+from pylhc_submitter.constants.external_paths import MADX_BIN, SIXDESK_UTILS
 
 # Program Paths ----------------------------------------------------------------
-SETENV_SH = SIXDESK_UTILS / "set_env.sh"
-MAD_TO_SIXTRACK_SH = SIXDESK_UTILS / "mad6t.sh"
-RUNSIX_SH = SIXDESK_UTILS / "run_six.sh"
-RUNSTATUS_SH = SIXDESK_UTILS / "run_status"
-DOT_PROFILE = SIXDESK_UTILS / "dot_profile"
-SIXDB = SIXDESK_UTILS.parent / "externals" / "SixDeskDB" / "sixdb"
+UTILITIES_DIR = Path("utilities")

```

Comment:
Annoying personal opinion: I would type hint these so that anyone importing them has their IDE inform them they're `Path`s